### PR TITLE
[FIX] test_themes: patch website switcher template

### PR DESCRIPTION
**Pull request by bso-odoo**
In [1] when all modules were migrated to the new assets system, the modules in design themes were forgotten.

This commit puts the owl templates into the correct asset.

[1]: https://github.com/odoo/odoo/commit/5410b7c238f211ced2519e53afc99c8468e1f527

### CODE REVIEW

**Review by rdeodoo — CHANGES_REQUESTED** [review]
I doubt you are referring to the correct commit, the [commit you mention](https://github.com/odoo/odoo/commit/03641610c2b4a6b832ea87795fd21e1d7af10b59) is a commit of april 2021.

What you want to refer is probably the recent one of @Gorash which removed the qweb assets a few days ago.

It just broke very recently, we fixed it after frontend > backend with https://github.com/odoo/design-themes/commit/fb23f19c1bcf861ae63573d435f6dbdf4b6d6a64

File: test_themes/__manifest__.py [hunk 1]
```diff
@@ -48,7 +48,7 @@
         'web.assets_backend': [
             'test_themes/static/src/js/systray_items/website_switcher.js',
         ],
-        'web.assets_qweb': [
+        'website.assets_editor': [
```
Comment (rdeodoo) [inline test_themes/__manifest__.py:51]: Any reason not to have both the xml and js file in the same bundle, like
```py
'web.assets_editor': [
    'test_themes/static/src/js/systray_items/*',
],
```
And maybe you can also remove the `js` folder to fit the community structure ?

**Review by rdeodoo — CHANGES_REQUESTED** [review]

File: test_themes/__manifest__.py (hunk 1 above)
Comment (bso-odoo) [reply test_themes/__manifest__.py:48]: @rdeodoo Change done.

Comment (rdeodoo) [reply test_themes/__manifest__.py:48]: Thank you, it was a bit nitpicking but at least now it is aligned with the rest :)

**Review by rdeodoo — APPROVED** [review]
@robodoo r+
Tested with all user rights :+1:

**Pull request closed without merging**